Packs/SymantecDLP/Integrations/SymantecDLPV2/SymantecDLPV2.py

```diff
@@ -0,0 +1,909 @@
+    """
+    Performs the fetch incidents functionality, which means that every minute if fetches incidents
+    from Symantec DLP and uploads them to Cortex XSOAR server.
+    :param client: Cortex XSOAR Client
+    :param fetch_time: For the first time the integration is enabled with the fetch incidents functionality, the fetch
+    time indicates from what time to start fetching existing incidents in Symantec DLP system.
+    :param fetch_limit: Indicates how many incidents to fetch every minute
+    :param last_run: Cortex XSOAR last run object
+    :param incident_types: The incident type to filter.
+    :param incident_status_id: The incident status ID to filter.
+    :param incident_severities: The incident severities to filter.
+    :return: A list of Cortex XSOAR incidents
+    """
+    if last_run and last_run.get('last_incident_creation_time'):
+        last_update_time = last_run.get('last_incident_creation_time')
+        last_incident_id = last_run.get('last_incident_id', '')
+
+    else:
+        # In first run
+        last_update_time = parse_creation_date(fetch_time)
+        last_incident_id = ''
+
+    incidents = []
+    if incident_severities:
+        incident_severities = [INCIDENT_SEVERITY_MAPPING[severity] for severity in incident_severities]  # type: ignore
+    if incident_types:
+        incident_types = [INCIDENT_TYPE_MAPPING[incident_type] for incident_type in incident_types]
+    incidents_data_list = client.get_incidents_request(creation_date=last_update_time, status_id=incident_status_id,
+                                                       severity=incident_severities,  # type: ignore
+                                                       incident_type=incident_types, limit=fetch_limit, order_by="ASC")
+    incidents_data_list = incidents_data_list.get('incidents', [])
+    if incidents_data_list:
+
+        for incident_data in incidents_data_list:
+            incident_id = incident_data.get('incidentId')
+            incident_creation_time = (dateparser.parse(incident_data.get('creationDate', ''))).strftime(
+                '%Y-%m-%dT%H:%M:%SZ')
+            if is_incident_already_fetched_in_previous_fetch(last_update_time, incident_creation_time, last_incident_id,
+                                                             incident_id):
+                # Skipping last incident from last cycle if fetched again
+                continue
+
+            incident_details = get_incident_details_fetch(client, incident_data)
+            incident: dict = {
+                'rawJSON': json.dumps(incident_details),
+                'name': f'Symantec DLP Incident ID {incident_id}',
+                'occurred': incident_creation_time
+            }
+            incidents.append(incident)
+            if incident_id == incidents_data_list[-1].get('incidentId'):
+                last_update_time = incident_creation_time
+                last_incident_id = incident_id
```

Comment:
Why do we need both the time and the ID if we only compare the ID?
Let's discuss this logic as a whole flow